packages/report-server/src/reportBuilder/transform/transform.ts

```diff
@@ -68,13 +78,13 @@ const buildParams = (params: unknown): TransformParams => {
   return {
     title,
     name: transformStep,
-    apply: transformBuilders[transformStep](restOfTransformParams),
+    apply: transformBuilders[transformStep](restOfTransformParams, context as Context),
```

Comment:
Is this `as Context` required? If it being optional is an issue perhaps lets just provide a default empty context?

Reply:
That's a bit tricky. My understandin is that it is required in some non-aliased transforms, which makes it generally required in their "type". At this code path we are calling a non-aliased transform.

Happy to have a call and see if there is a better way around that!

Reply:
I think the general approach with validation in report-server has been that the higher levels make type assertions to the lowest common denominator, and then it's up to the children to validate up to their required level. 

In this instance I guess that would mean that context can be optional in this scope, as it's not required by all. And then for the transforms that do require it there's a secondary validation check to assert it is defined?

Yup, happy to call on this if you want 👍